api/envoy/config/grpc_credential/v2alpha/BUILD

```diff
@@ -1,6 +1,6 @@
 licenses(["notice"])  # Apache 2
 
-load("//bazel:api_build_system.bzl", "api_proto_library", "api_go_proto_library")
+load("//bazel:api_build_system.bzl", "api_go_proto_library", "api_proto_library")
```

Comment:
the old buildifier doesn't enforce the order so this passes check, new buildifier enforces the order.